services/game-actions.js

```diff
@@ -0,0 +1,37 @@
+import firebase from 'firebase'
+import moment from 'moment';
+
+export function createGame(game) {
+  return new Promise((resolve, reject) => {
+    firebase.database().ref(`games/${game.id}`).set({
+      creator: "-LCF7qa2LI_oKKSQ3SIA",
+      id: game.id,
+      place: game.place,
+      year: game.year,
+      timestamp: moment.now(),
+    })
+    resolve(true)
+  })
+}
+
+export function generateGameKey() {
+  return new Promise((resolve, reject) => {
+    var key = firebase.database().ref().child('games').push().key
+    resolve(key)
+  })
+}
+
+export function getGame(game) {
+  return new Promise((resolve, reject) => {
+    firebase.database().ref(`games/${game.id}`).once('value').then(snapshot => {
+      resolve(snapshot.val())
+    });
+  })
+}
+
+export function deleteGame(game) {
+  return new Promise((resolve, reject) => {
+    firebase.database().ref(`games/${game.id}`).remove()
+    })
+    resolve(true)
+}
```

Comment:
Add an `export function updateGame(id, newGame)` function that when given an id of a game, you reassign **all** the properties of the old game with the newGame

Reply:
Can't I just set new properties within the old game?

Reply:
Yeah, technically you could. However, it's best practice to create functions, especially functions interacting with a database, that are clearly marked with a purpose. It makes sense to have `createGame` because you want to create a game. However, it doesn't make much sense if you're using the same function to update a game. That's why you would want to have `updateGame` so it's clear to you, and other developers, what the purpose of that function is.